app/assets/stylesheets/admin/openfoodnetwork.css.scss

```diff
@@ -2,6 +2,7 @@
 
 input[type="submit"], input[type="button"], button, .button {
   cursor: pointer;
+  padding: 6.5px 15px;
```

Comment:
this is adding padding to all buttons in the backoffice right? I havent seen how it looks... are you sure this is what we need? We may need to test more pages to verify this is not breaking the structure in any other page.

Reply:
I had a quick look around, and yes this looks right. The font has changed for the backoffice, and the new one renders using slightly taller characters, which means the overall height of all buttons has increased slightly, making them look strange next to some of the other elements that have a fixed height that hasn't increased. This adjusts the padding a _tiny_ bit so they look like they did before and fit in visually with other elements.

Reply:
awesome!